tests/test_server.py

```
@@ -434,27 +448,79 @@ def test_create_ticket_tool(mock_zammad_client, ticket_factory):
     mock_instance.add_article.assert_called_once_with(
-        ticket_id=1, body="New comment", article_type="note", internal=False
+        ticket_id=1, article_type="note", body="New comment", internal=False, sender="Agent"
     )
 
 
+def test_add_article_invalid_type():
+    """Test that ArticleCreate rejects invalid article types."""
+    from mcp_zammad.models import ArticleCreate
+
+    # Test invalid article type
+    with pytest.raises(Exception) as exc_info:  # Pydantic ValidationError
```

Ruff (0.14.1)

`pytest.raises(Exception)` is too broad, set the `match` parameter or use a more specific exception

(PT011)